tidy3d/components/autograd/derivative_utils.py

```diff
@@ -70,6 +70,12 @@ class DerivativeInfo(Tidy3dBaseModel):
         description="Bounds corresponding to the structure, used in ``Medium`` calculations.",
     )
 
+    frequency: float = pd.Field(
+        ...,
+        title="Frequency of adjoint simulation",
+        description="Frequency at which the adjoint gradient is computed.",
+    )
+
```

Comment:
Does this imply that we'll have a separate `DerivativeInfo` for each frequency?

Reply:
No, in the multi-frequency branch, I will change this to a list of frequencies. for now it's just single, though.